src/globus_sdk/config/environments.py

```diff
@@ -68,6 +83,17 @@ def get_service_url(environment: str, service: str) -> str:
     return url
 
 
+def get_webapp_url(environment: Optional[str] = None) -> str:
+    """
+    Return the URL to access the Globus web app in the given environment. For example:
+
+    >>> get_webapp_url("sandbox")
+    'https://app.sandbox.globuscs.info/'
```

Comment:
```suggestion
    >>> get_webapp_url("preview")
    'https://app.preview.globus.org/'
```

As above, using `preview` or `production` only.